### PR TITLE
Adds ghost visible countdowns for bombs

### PR DESCRIPTION
**Pull request by coiax**
:cl: coiax
rscadd: For extra observer drama, ghosts can now visibly see the
countdown of syndicate bombs, nuclear devices, cloning pods,
gang dominators and AI doomsday devices.
/:cl:
- Also updates my build script to take account of the move of dm.sh

image

### CODE REVIEW

File: code/modules/countdown/countdown.dm [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+/obj/effect/countdown/process()
+	if(!attached_to || qdeleted(attached_to))
+		qdel(src)
+	var/new_val = get_value()
+	if(new_val == last_displayed)
+		return
+	last_displayed = new_val
+	if(new_val)
+		var/image/text_image = new(loc = src)
+		text_image.maptext = "<font size=4>[new_val]</font>"
```
Comment (RemieRichards) [inline code/modules/countdown/countdown.dm]: font size should be an arg
```"<font size = [fontsize]">```

**Comment by Shadowlight213** [comment]
Ehh. It seems like it would look dumb for cloning pods and stuff.
Plus outside of the nuke, you can already examine them to find the countdown

**Comment by coiax** [comment]
It's not providing information they don't have access to already, it just means that it's visible without effort, meaning you can all appreciate the countdown that you can all reliably see.

**Comment by ChangelingRain** [comment]
Fancy.

**Pull request merged**